azext_iot/central/params.py

```diff
@@ -212,15 +227,40 @@ def load_central_arguments(self, _):
             "by visiting https://github.com/iot-for-all/iot-central-high-availability-clients#readme",
         )
 
+    with self.argument_context("iot central device twin") as context:
+        context.argument(
```

Comment:
@HangyiWang are you using --replace under `update` command or use `replace` as command?

Reply:
Talked with @vilit1 last week. Will use `device replace` command but not `device update -r`.